Develop

I-ra opened the pull request:
The project for the 4th sprint was written

I-ra committed:
The project for the 4th sprint was written

I-ra committed:
The project for the 4th sprint was written

I-ra committed:
The project for the 4th sprint was written

I-ra committed:
The project for the 4th sprint was written

I-ra committed:
The project for the 4th sprint was written

pomegranate282922 commented on src/main/java/yandex/praktikum/second_project/pages/HomePage.java:
```diff
@@ -0,0 +1,75 @@
+package yandex.praktikum.second_project.pages;
```
В пакетах _ не используем 
Можно улучшить

pomegranate282922 commented on src/main/java/yandex/praktikum/second_project/pages/HomePage.java:
```diff
@@ -0,0 +1,75 @@
+
+
+    public String getAnswerTextByIndex(int questionIndex) throws InterruptedException {
+        Thread.sleep(1000);
```
Все слипы нужно заменить на явные ожидания 
Нужно исправить

pomegranate282922 commented on src/main/java/yandex/praktikum/second_project/pages/OrderData.java:
```diff
@@ -0,0 +1,65 @@
+    public String getComment() {
+        return comment;
+    }
+}
```
Отлично что введен pojo для данных

pomegranate282922 commented on src/test/java/yandex/praktikum/second_project/pages/AccordionTest.java:
```diff
@@ -0,0 +1,66 @@
+    public void setup() {
+        driver = new ChromeDriver();
+        homePage = new HomePage(driver);
+        driver.get("https://qa-scooter.praktikum-services.ru/");
```
Урлы лучше вынести как константы 
Можно улучшить

pomegranate282922 commented on src/test/java/yandex/praktikum/second_project/pages/AccordionTest.java:
```diff
@@ -0,0 +1,66 @@
+
+    @After
+    public void teardown() {
+        driver.quit();
```
Общие начала и концы всех тестовых классов удобно вынести в общий класс предок 
Можно улучшить

pomegranate282922 commented on src/test/java/yandex/praktikum/second_project/pages/OrderTest.java:
```diff
@@ -0,0 +1,73 @@
+        mainPage = new HomePage(driver);
+        orderPage = new OrderPage(driver);
+        driver.get("https://qa-scooter.praktikum-services.ru/");
+        mainPage.getCoocie();
```
cookie

pomegranate282922 reviewed (commented):
Очень хорошая работа! Все тесты реализованы. Применена параметризация.
Замечания: убрать все слипы, заменив из на явные ожидания

I-ra committed:
Поправлено наименование пакета
Слипы заменены на явные ожидания
URL вынесен как константа
Общие начала и концы всех тестовых классов вынесены в общий класс предок
Поправлено наименование метода getCookie()

I-ra committed:
Поправлено наименование пакета
Слипы заменены на явные ожидания
URL вынесен как константа
Общие начала и концы всех тестовых классов вынесены в общий класс предок
Поправлено наименование метода getCookie()

pomegranate282922 reviewed (approved):
отлично, зачет!
